index.html

```diff
@@ -0,0 +1,237 @@
+            <h2 class="card d-flex">
+                <img src="Snapshoot Portfolio.png">
+                <h1 class="works-primary-text"
+                >Tonic
+                </h1>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="counter"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="counter"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button type="button">See Project</button>
+            </h2>
+        </h2>
+        <h2 class="card-container app-bckground-color">
+            <h2 class="card d-flex">
+                <img src="Snapshoot Portfolio 2.png" alt="Card Image">
+                <h1 class="works-primary-text"
+                >Multi-Post <br/> Stories 
+                </h1>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button type="button">See Project</button>
+            </h2>
+        </h2>
+        <h2 class="card-container app-bckground-color">
+            <h2 class="card d-flex">
+                <img src="Snapshoot Portfolio 3.png">
+                <h1 class="works-primary-text"
+                >Tonic
+                </h1>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button type="button">See Project</button>
```

Comment:
- [ ] Please ensure that you style your `button` according to the given design requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=hnNtRADBdqYtrcFO-0) and `Implement the button interactions (enable, hover, etc.)`. :slightly_smiling_face: